requirements.txt

```diff
@@ -6,4 +6,5 @@ tabulate==0.9.0
 networkx==3.1
 graphviz
 matplotlib==3.7.1
+transformers==4.38.0
```

Comment:
this is only needed for dev, right? As users of the lib don't need it, lets move?

Reply:
Done.